src/python/grpcio/grpc/_cython/_cygrpc/aio/completion_queue.pxd.pxi

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cdef class BaseCompletionQueue:
+
+    cdef grpc_completion_queue* c_ptr(self)
+
+cdef class PollerCompletionQueue(BaseCompletionQueue):
+    cdef grpc_completion_queue *_cq
```

Comment:
can be this attribute moved to the base class?

Reply:
Done. I was hoping this could be an implementation detail, but I don't see other alternatives for now. So, moved.